src/inline.d

```diff
@@ -1207,7 +1207,11 @@ public:
              * Note that inline scanning of s.exp.e1 and s.exp.arguments was already done.
              */
             if (s.exp && s.exp.op == TOKcall)
-                visitCallExp(cast(CallExp)s.exp, null, true);
+            {
+                CallExp ce = cast(CallExp)s.exp;
+                if (ce.e1.op == TOKvar)     // workaround for Bugzilla 15296
+                    visitCallExp(ce, null, true);
```

Comment:
How does restricting the inlining to a specific case help to fix an issue where something is __not__ inlined.
Also `visitCallExp` already handles TOKvar as first branch (see [here](https://github.com/D-Programming-Language/dmd/blob/d167f328a011000227083d893d58305d78fc34e8/src/inline.d#L1498)).
The comment isn't useful either b/c the bugzilla page doesn't contain any information other than the bug report.
Please don't introduce obscure special cases w/o any explanation at all.